experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/list.spec.ts

```diff
@@ -716,7 +713,7 @@ describe("List", () => {
 	// TODO: Post-MVP
 
 	describe("implements T[]", () => {
-		describe("Setting [index: number] is disallowed (for MVP)", () => {
+		it("Setting [index: number] is disallowed (for MVP)", () => {
```

Comment:
@DLehenbauer These tests caught a bug for me! They are the only tests we had that inserted "nothing" into a list. I updated some of our list tests to add more coverage for this.

Reply:
...and thank you for fixing 'it'.  :-)